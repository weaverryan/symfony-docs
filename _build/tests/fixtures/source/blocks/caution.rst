.. caution::

    Using too many sidebars or caution directives can be distracting!
