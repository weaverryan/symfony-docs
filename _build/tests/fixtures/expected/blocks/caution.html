<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-wrapper">
    <div class="caution"></div>
    <div class="admonition admonition-caution">
        <p class="first admonition-title">Caution</p>
        <p class="last">Using too many sidebars or caution directives can be distracting!</p>
    </div>
</div>
</body>
</html>
